src/components/Breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { Meta, StoryObj } from '@storybook/react'
+import { cloneDeep, set } from 'lodash-es'
+import { useCallback, useMemo, useState } from 'react'
+import { action } from '@storybook/addon-actions'
+
+import { collapsedProps, controlledProps, defaultProps, loadingProps, uncontrolledProps, withMenuProps, withoutMenuProps } from './Breadcrumb.mocks'
+import { Breadcrumb } from './Breadcrumb'
+import { BreadcrumbItemMenu } from './Breadcrumb.types'
+
+const meta = {
+  component: Breadcrumb,
+} satisfies Meta<typeof Breadcrumb>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
```

Comment:
When running the storybook, it will display a code sample for each story, since I've seen some stories feature an unintelligible codeblock. How are these stories code samples displayed? are they usable from a storybook reader?

Reply:
Unfortunately storybook does not infer stories arguments if `decorators` are used (which is needed to properly open dropdowns). We have the same problem in the Modal component stories. We need to either review the pattern of having mocks in a separate file, or find a way to better tell storybook how the code blocks are made

Reply:
can you plesase open an issue on this matter?

Reply:
Done: https://github.com/mia-platform/design-system/issues/503